libs/newsletter/src/effects/newsletter.effects.ts

```diff
@@ -27,6 +30,14 @@ export class DaffNewsletterEffects<T extends DaffNewsletterSubmission, V>{
           return of(new DaffNewsletterFailedSubscribe('Failed to subscribe to newsletter'));
         })
       )
-    ))
-  )
+    )));
+
+  cancelSubmission$: Observable<Action> = createEffect(() => this.actions$.pipe(
```

Comment:
This needs to cancel the `trySubmission$` observable, it won't work as a standalone observable. This definitely needs test coverable a marble diagram. 

https://github.com/ReactiveX/rxjs/blob/master/docs_app/content/guide/testing/marble-testing.md
https://www.learnrxjs.io/operators/transformation/switchmap.html